### DISCUSSION TITLE
Allow in-page link to headings in shared docs

### DISCUSSION THREAD

**Discussion by furilo** [outline/outline#2651] [Ideas]
We want to be able to link to headings inside a page and that they worked on a shared page. But if we add a link only with the `#link_to_heading`:

<img width="878" alt="CleanShot 2021-10-08 at 11 56 10@2x" src="https://user-images.githubusercontent.com/24740/136536993-eb0ec9ab-c33f-40e1-80a7-e51f47960365.png">

when viewing the shared public page and clicking on one of this links you get to a blank screen, while the desired behaviour would be a scroll-to to the corresponding header. 

The link seems correctly added (example: https://populate.getoutline.com/share/c091ee4d-ad76-4857-b5d1-cf3b9bd23ead#h-busqueda )

<img width="1368" alt="CleanShot 2021-10-08 at 13 24 21@2x" src="https://user-images.githubusercontent.com/24740/136548560-dcd8f2e2-35b7-46cd-b488-72cef5efb876.png">

**Comment by tommoor** [comment]
I created an issue from this discussion, it seems like a bug.
